src/se3/lie.hpp

```diff
@@ -36,17 +52,15 @@ class SE3 {
 
     SE3() = default;
 
-    SE3(Eigen::Vector3d position, Eigen::Quaterniond const& rotation);
+    explicit SE3(R3 position, SO3 rotation = SO3{Eigen::Quaterniond::Identity()});
```

Comment:
shouldn't you just use SO3::Identity() here?

Reply:
Fixed, default ctor for SO3 is identity now